assets/src/edit-story/components/library/text/fontPreview.js

```diff
@@ -40,6 +40,7 @@ const Preview = styled.button`
   width: 100%;
   border: none;
   cursor: pointer;
+  text-align: left;
```

Comment:
Below in this file there is `fontWeight={fontWeight}` used for the preview font-weight, however, `fontWeight` is actually `undefined` -- the text presets have inline font-weight only. This results in the preset previews (influences the heading 1 mainly) not having the correct weight as seen in Figma:
<img width="331" alt="Screenshot 2020-08-27 at 13 00 50" src="https://user-images.githubusercontent.com/3294597/91427071-abca8400-e85d-11ea-8f0e-d5df5a4c2bd9.png">

Reply:
Good call out!  I updated the presets to have font-weight and now the buttons render correctly:
<img src="https://user-images.githubusercontent.com/40646372/91476051-4c667700-e851-11ea-956a-56acb6362ea4.png" height="300">